test/browser/column/key.test.ts

```diff
@@ -10,6 +10,6 @@ test('key is escaped in query selectors', () => {
   ];
 
   expect(() => {
-    setup({ columns, rows: [] });
+    setupNew({ columns, rows: [] });
   }).not.toThrow();
```

Comment:
Does it throw if we don't escape the key?

Reply:
It does 

![image](https://github.com/user-attachments/assets/36dee709-11ab-4084-9588-9bfe41e40db8)